.github/workflows/dependency-review.yaml

```diff
@@ -21,6 +21,7 @@ jobs:
         pkg:cargo/ring,
         pkg:cargo/sspi,
         pkg:cargo/tokio-boring,
+        pkg:cargo/tokio-rustls,
```

Comment:
`tokio-rustls` uses `MIT/Apache-2.0` but dependecy review action can't handle slashes

Reply:
Some yes (`curve25519-dalek-derive`), some should work with vanilla dependency review rules (most are MIT or Apache). I'll revisit it in separate PR - we want to merge this one ASAP